features/cli/build.feature

```diff
@@ -23,6 +23,9 @@ Feature: build 'apps' with CLI
     Then the output should contain:
       | ruby-hello-world |
     When I run the :new_build client command with:
+      | app_repo |  openshift/ruby:latest~https://github.com/openshift/ruby-hello-world.git |
+      | strategy | docker                                                                   |
+      | name     | n1                                                                       |
```

Comment:
I saw above 3 lines are duplicated with beleow 3 lines?

Reply:
Updated, I think now you are happy.

Reply:
@wewang58 , does PR look good for you now.

Reply:
yes, LGTM